pwnlib/data/templates/pwnup.mako

```diff
@@ -131,13 +131,12 @@ def start(argv=[], *a, **kw):
 %endif
 gdbscript = '''
 %if ctx.binary:
+set sysroot
   %if 'main' in ctx.binary.symbols:
-break *0x{exe.symbols.main:x}
-  %else:
-break *0x{exe.entry:x}
```

Comment:
This needs to remain.  In the event symbols are unavailable, `gdbserver` will break at the entry point to the *linker*, not the entry point to the *main binary*.

In the event that the binary is PIE (and `exe.entry` is a small value ~close to zero), this has the same effect as setting a breakpoint on the first instruction because of a convenient bug in `gdb`: https://reverseengineering.stackexchange.com/q/8724

Reply:
Setting an invalid breakpoint in the gdbscript appears to be an error:

```
0x00007f9dc32aefb0 in _start () from target:/lib64/ld-linux-x86-64.so.2
Temporary breakpoint 1 at 0x520
Warning:
Cannot insert breakpoint 1.
Cannot access memory at address 0x520

/tmp/pwnrsyWx3.gdb:8: Error in sourced command file:
Command aborted.
```

I still end up in `_start` and am unable to continue the program from there:

```
gef➤  continue
Continuing.
Warning:
Cannot insert breakpoint 1.
Cannot access memory at address 0x520

Command aborted.
```